components/Frame/Popover/UserNav.js

```diff
@@ -0,0 +1,279 @@
+                        active={active}
+                        closeHandler={closeHandler}
+                      >
+                        {t('nav/give')}
+                      </NavLink>
+                      <NavLink
+                        route='legal/imprint'
+                        active={active}
+                        closeHandler={closeHandler}
+                      >
+                        {t('nav/team')}
+                      </NavLink>
+                      <NavLink
+                        {...fontStyles.sansSerifLight16}
+                        route='pledge'
+                        params={{ package: 'DONATE' }}
+                        active={active}
+                        closeHandler={closeHandler}
+                      >
+                        {t('nav/donate')}
+                      </NavLink>
+                    </div>
+                  </div>
+                  <div {...styles.navSection}>
+                    <div {...styles.navLinks} {...styles.smallLinks}>
+                      <SignOut Link={SignoutLink} />
+                    </div>
+                  </div>
+                </>
+              )}
+            </>
+          )}
+          {inNativeApp && hasExpandedRef.current && <Footer />}
+        </div>
+      </Center>
+    </>
+  )
+}
+
+const styles = {
+  container: css({
+    [mediaQueries.mUp]: {
+      marginTop: '40px'
+    }
+  }),
+  hr: css({
+    margin: 0,
+    display: 'block',
+    border: 0,
+    height: 1,
+    color: colors.divider,
+    backgroundColor: colors.divider,
+    width: '100%'
+  }),
+  hrFixed: css({
+    position: 'fixed',
+    top: HEADER_HEIGHT_MOBILE - 1,
+    [mediaQueries.mUp]: {
+      top: HEADER_HEIGHT - 1
+    }
+  }),
+  signInBlock: css({
+    display: 'block'
+  }),
+  bookmarkContainer: css({
+    width: '100vw',
+    position: 'relative',
+    left: '50%',
+    right: '50%',
+    marginLeft: '-50vw',
+    marginRight: '-50vw'
+  }),
+  navSection: css({
+    display: 'flex',
+    flexDirection: 'column',
+    margin: '24px 0px'
+  }),
+  navLinks: css({
+    display: 'flex',
+    flexDirection: 'column',
+    width: '100%',
+    [mediaQueries.mUp]: {
+      flexDirection: 'row'
+    },
+    '& a': {
+      flexShrink: 0,
+      ...fontStyles.sansSerifMedium20,
+      [mediaQueries.mUp]: {
+        ...fontStyles.sansSerifMedium22,
+        marginRight: 36
+      }
+    },
+    '& a:not(:last-child)': {
+      marginBottom: 24,
+      [mediaQueries.mUp]: {
+        marginBottom: 0
+      }
+    }
+  }),
+  regularLinks: css({
+    '& a': {
+      ...fontStyles.sansSerifRegular,
+      fontSize: 20,
+      lineHeight: '24px',
+      letterSpacing: 'normal',
+      [mediaQueries.mUp]: {
+        ...fontStyles.sansSerifRegular22
+      }
+    }
+  }),
+  smallLinks: css({
+    '& a': {
+      ...fontStyles.sansSerifRegular18
+    }
+  })
+}
+
+export default compose(
+  withT,
+  graphql(notificationsQuery),
```

Comment:
There is a loading explosion on love because loading and error state of this query aren't handled.

Move this to `NotificationFeedMini` and properly handle the loading state there (with a `<Loader />` like book mark mini feed, btw it's once `MiniFeed` and once `FeedMini`, I'm in favor of mini feed)

Reply:
done 🧞